ambry-rest/src/main/java/com/github/ambry/rest/NettyResponseChannel.java

```diff
@@ -123,6 +123,7 @@
     public void operationComplete(ChannelFuture future) {
       logger.trace("scheduling closure of channel {}", future.channel());
       future.channel().eventLoop().schedule(() -> {
+        nettyMetrics.delayedCloseExpiredCount.inc();
```

Comment:
Nit: nothing has really expired here.  Suggest renaming this delayedCloseActivatedCount or delayedCloseTriggerCount.

Reply:
changed the metric name to delayedCloseActivatedCount. Thanks